dbt/include/athena/macros/adapters/schema.sql

```diff
@@ -0,0 +1,12 @@
+{% macro default__create_schema(relation) -%}
```

Comment:
Shouldn't this be `athena__` prefix instead?

Reply:
Yes indeed I did not check all the code in the previous commits :)